hw/ip/clkmgr/data/clkmgr.hjson.tpl

```diff
@@ -187,11 +187,16 @@
           name: "SEL",
           mubi: true,
           desc: '''
-            A value of kMultiBitBool4True selects external clock as clock for the system.
-            While this register can always be programmed, it only takes effect when the system is in
-            life cycle TEST or RMA states when DFT is enabled.
+            When current value is not kMultiBitBool4True, a value of kMultiBitBool4True selects
+            external clock as clock for the system.  Any other value during this state has
+            no impact.
 
-            All other values are invalid and keep clocks on internal sources.
+            When current value is kMultiBitBool4True, a value of kMultiBitBool4False selects
+            internal clock as clock for the system. Any other value during this stage has no
+            impact.
+
+            While this register can always be programmed, it only takes effect when the system
+            is in life cycle TEST, DEV or RMA states when hardware debug is enabled.
```

Comment:
https://docs.opentitan.org/doc/security/specs/device_life_cycle/ seems to say dft is not enabled in DEV state, so

```suggestion

            While this register can always be programmed, it only takes effect when DFT functions
            are enabled, which is in life cycle TEST_UNLOCKED or RMA states.
```